lib/ansible/utils/collection_loader.py

```diff
@@ -116,6 +116,29 @@ def load_module(self, fullname):
 
         return mod
 
+    def get_source(self, fullname):
+        mod = sys.modules.get(fullname)
+        if not mod:
+            mod = self.load_module(fullname)
```

Comment:
This block is repeated in two methods, it deserves to be put into a separate helper method.